addon/components/bs4/bs-navbar.js

```diff
@@ -4,7 +4,19 @@ import Navbar from 'ember-bootstrap/components/base/bs-navbar';
 export default Navbar.extend({
   classNameBindings: ['breakpointClass', 'backgroundClass'],
 
-  type: 'light',
+  type: computed('appliedType', {
+    get() {
+      return this.get('appliedType');
+    },
+
+    set(key, value) { // eslint-disable-line no-unused
+      let newValue = (!value || value === 'default') ? 'light' : value;
+      this.set('appliedType', newValue);
+      return newValue;
+    }
+  }),
+
+  appliedType: 'light',
```

Comment:
Why is this separate property needed?

Reply:
That was my first approach, but I needed the setter to redirect the set anyway. I'll look at it again and see if it can be reduced or simplified.